runner.rb

```diff
@@ -0,0 +1,3 @@
+require './lib/cell'
+require './lib/grid'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.